docs/reference/api/docker_remote_api.md

```diff
@@ -130,6 +130,7 @@ This section lists each version from latest to oldest.  Each listing includes a
   containers that are tasks (part of a service in swarm mode).
 * `POST /containers/create` now takes `StopTimeout` field.
 * `POST /services/create` and `POST /services/(id or name)/update` now accept `Monitor` and `MaxFailureRatio` parameters, which control the response to failures during service updates.
+* `GET /networks/(name)` now returns `Created`.
```

Comment:
@allencloud yes, looking at it again, the description may be a bit short; perhaps you could open a PR to improve it?